<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartViewer</class>
 <widget class="QDialog" name="ChartViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wykres</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCustomPlot" name="customPlot" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>Src/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
